src/scss/normalize.css

```diff
@@ -163,7 +139,6 @@ select,
 textarea {
   font-family: inherit; /* 1 */
   font-size: 100%; /* 1 */
-  line-height: 1.15; /* 1 */
```

Comment:
Same question here as on https://github.com/COVID19Tracking/website/pull/427/files#r405601317, I guess! Was this line-height creating some issue we couldn’t override?